src/nvim/buffer_defs.h

```diff
@@ -1126,6 +1126,9 @@ struct window_S {
                                     // time through cursupdate() to the
                                     // current virtual column
 
+  linenr_T w_cursorline;            ///< Where 'cursorline' should be drawn,
+                                    ///  can be different from w_cursor.lnum
+                                    ///  for closed folds.
```

Comment:
```suggestion
                                    ///< can be different from w_cursor.lnum
                                    ///< for closed folds.
```
There was also a mistake in the code above:
```diff
diff --git a/src/nvim/buffer_defs.h b/src/nvim/buffer_defs.h
index 3601dc706..6669e2303 100644
--- a/src/nvim/buffer_defs.h
+++ b/src/nvim/buffer_defs.h
@@ -1112,7 +1112,7 @@ struct window_S {
   win_T *w_prev;              ///< link to previous window
   win_T *w_next;              ///< link to next window
   bool w_closing;                   ///< window is being closed, don't let
-                                    ///  autocommands close it too.
+                                    ///< autocommands close it too.
 
   frame_T *w_frame;             ///< frame containing this window
 
```